camel/logger.py

```diff
@@ -0,0 +1,102 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import logging
+import os
+import sys
+
+# Create a private logger
+_logger = logging.getLogger('camel')
+_logging_disabled = False
+
+
+class PrintLogger(logging.Logger):
+    def _log(self, level, msg, args, **kwargs):
+        if args:
+            msg = msg + ' ' + ' '.join(str(arg) for arg in args)
+        super()._log(level, msg, (), **kwargs)
```

Comment:
can we use logging.Logger or logging.Formatter instead?

Reply:
it is calling logging.Logger._log, the purpose of this is to allow to pass args in the way print would. Do you think it would be better to remove this and change the callers to call in a more standard logging.Logger format?

Reply:
Yeah, I feel using the standard logging.Logger format could be more maintainable and consistent with common logging practices, WDYT?

Below is some code FYR
```
# Standard logger setup
logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)

# Create and configure console handler
console_handler = logging.StreamHandler()
console_handler.setLevel(logging.DEBUG)

# Create and set formatter for the handler
formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
console_handler.setFormatter(formatter)

# Add handler to the logger
logger.addHandler(console_handler)
```

Reply:
Makes sense. I think a reason I did things this printlogger way was that I was expecting there to be more usages and it would be quite a task to port everything, but actually there aren't so many prints in the codebase so it's not so bad to change those places